### ISSUE TITLE
Where does netopeer2-cli get the public host key of netopeer2-server ?

### ISSUE THREAD

**Issue by ezekieltsui** [CESNET/netopeer2#826]
Generally, we (`ssh-client`) use `ssh` command to login the remote host (`ssh-server`). If it's the first time we connect, `ssh-server` should send its `public host key` that exists in `/etc/ssh` to us, then we `ssh-client` should save the `public host key` in a file called `known_hosts`.

In netopeer scenario, we think of `netopeer2-cli` as `ssh-client`, and `netopeer2-server` as `ssh-server`. After using `netopeer2-cli` to login in to `netopeer2-server`, a public host key record was added to `known_hosts`. But the `public host key in known_hosts file on client side` doesn's match the `public host key in /etc/ssh on server side`, nor `the originally configured public key in ietf-netconf-server on server side`. 

So, where does the public host key in `known_hosts` come from ? And what is the correspondence between keys ? Looking forward to your response to my question, thx very much.

**Comment by michalvasko** [comment]
If you look carefully at the output of `# make install` of *netopeer2*, you should get some idea. In short, [this](https://github.com/CESNET/netopeer2/blob/master/scripts/merge_hostkey.sh) script is being executed. You can print the generated key using `sysrepocfg -X -m ietf-keystore` and are, naturally, free to modify the key configuration however you want. This is performed just so that you can connect right after installing the server.

**Comment by ezekieltsui** [comment]
> If you look carefully at the output of `# make install` of _netopeer2_, you should get some idea. In short, [this](https://github.com/CESNET/netopeer2/blob/master/scripts/merge_hostkey.sh) script is being executed. You can print the generated key using `sysrepocfg -X -m ietf-keystore` and are, naturally, free to modify the key configuration however you want. This is performed just so that you can connect right after installing the server.

In fact, I have used the `sysrepocfg -X -m ietf-keystore` command to get the netopeer server side's public key. I compared this server side's public key with the known_hosts of the netopeer client, they do not match.

The detailed process is as follows：
I started two virtual machines, one as netopeer server and the other as netopeer client. 
Netopeer server uses 192.168.109.166 as its ip address, and the public key in ietf-keystore is : `<name>genkey</name>
      <algorithm>rsa2048</algorithm>
      <public-key>MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA1gfv/w3c2DzNB3yRO9uW
Ghf2vLK/i/Z/ZXGyYgdx9UWep02LD6GjTRKdMnqa3s4EHH1tO8sPxMPDs/CKi0s4
zqu6NTvAZtJybTV84RWs1QS5QHRitdRq+BzFNS2w2rQ4ibBFXxd8jQtfwTCPuVFz
tD76kPhcCGei4U7D75jpXTcC8LlhrmG5IGCjonErH2MNfsdLAnNjRLC6JaH9bhbE
gPO1gRkkLpQuKUVbNphTDd9HM8YOUSilw5N66fc7SC29en7Qpu3VcB/xYun6ml92
WctNfLWg4JTTXANwnIfrmG/GLocg5fcSQleU9TI80VLCb+yAhsFNOncHbeBnhirP
OQIDAQAB</public-key>
`
And ietf-netconf-server references the above public key in the listen container.

On the other side, after the netopeer client verifies the fingerprint, a public key record (ssh-rsa) is generated in known_hosts.
`[192.168.109.166]:830 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDWB+//DdzYPM0HfJE725YaF/a8sr+L9n9lcbJiB3H1RZ6nTYsPoaNNEp0yeprezgQcfW07yw/Ew8Oz8IqLSzjOq7o1O8Bm0nJtNXzhFazVBLlAdGK11Gr4HMU1LbDatDiJsEVfF3yNC1/BMI+5UXO0PvqQ+FwIZ6LhTsPvmOldNwLwuWGuYbkgYKOicSsfYw1+x0sCc2NEsLolof1uFsSA87WBGSQulC4pRVs2mFMN30czxg5RKKXDk3rp9ztILb16ftCm7dVwH/Fi6fqaX3ZZy018taDglNNcA3Cch+uYb8YuhyDl9xJCV5T1MjzRUsJv7ICGwU06dwdt4GeGKs85`

This two public key do not match. So I don’t know the source of the known_hosts record, nor what the role of the public key in ietf-netconf-server is now.
